src/main/java/com/training/assignments/miniproject/Employee.java

```diff
@@ -0,0 +1,67 @@
+package com.training.assignments.miniproject;
+
+import lombok.Data;
+import java.util.ArrayList;
+import java.util.List;
+
+@Data
+public class Employee {
+    List<Employee> employeeList = new ArrayList<>();
+
+    String name;
+    String address;
+    int salary;
+    int id;
+
+    public Employee(String name, String address, int salary, int id) {
+        this.name = name;
+        this.address = address;
+        this.salary = salary;
+        this.id = id;
+    }
+
+    public Employee() {
+    }
+
+    public List<Employee> getCreatedEmployee() {
+        employeeList.add(new Employee("Shri", "kandivali", 20000, 56));
+        employeeList.add(new Employee("Ashish", "Malad", 25000, 84));
+        employeeList.add(new Employee("Arnav", "Goregaon", 3000, 75));
+        employeeList.add(new Employee("Paramveer", "Andheri", 40000, 420));
+        return employeeList;
+    }
```

Comment:
Why are we hardcoding values inside the class?
All the values should be passed from the test class.
@PranavHanwante @shrikanttechincal this is a poor design - use the design which you used in Student class.